storage/s3/storage.go

```diff
@@ -0,0 +1,22 @@
+package s3
+
+import (
+	"context"
+
+	"github.com/gin-gonic/gin"
+)
+
+const key = "s3"
+
+type S3Storage interface {
```

Comment:
We can use s3iface.S3API instead of this right?

Reply:
@FilipNikolovski can you please explain whats the difference